docs/tutorials/06-batch-infer.ipynb

```diff
@@ -695,10 +690,68 @@
    ],
    "source": [
     "project.enable_model_monitoring(\n",
-    "    base_period=1, wait_for_deployment=True, deploy_histogram_data_drift_app=True\n",
+    "    base_period=1, wait_for_deployment=True, deploy_histogram_data_drift_app=False\n",
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "63d483c2",
+   "metadata": {},
+   "source": [
+    "## Change the *histogram data drift* application defaults \n",
+    "\n",
+    "To generate the drift table plot artifact using MLRun's *histogram data drift* application, we have to change the application defaults."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d37e8132",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "> 2025-03-11 16:56:19,943 [debug] Mount already applied or auto-mount manually disabled - not performing auto-mount\n",
+      "> 2025-03-11 16:56:19,945 [info] Starting remote function deploy\n",
+      "2025-03-11 14:56:20  (info) Deploying function\n",
+      "2025-03-11 14:56:20  (info) Building\n",
+      "2025-03-11 14:56:20  (info) Staging files and preparing base images\n",
+      "2025-03-11 14:56:20  (warn) Using user provided base image, runtime interpreter version is provided by the base image\n",
+      "2025-03-11 14:56:20  (info) Building processor image\n",
+      "2025-03-11 14:58:05  (info) Build complete\n",
+      "2025-03-11 14:58:23  (info) Function deploy complete\n",
+      "> 2025-03-11 16:58:31,733 [info] Model endpoint creation task completed with state succeeded\n",
+      "> 2025-03-11 16:58:31,733 [info] Successfully deployed function: {\"external_invocation_urls\":[],\"internal_invocation_urls\":[\"nuclio-tutorial1-admin-histogram-data-drift.default-tenant.svc.cluster.local:8080\"]}\n",
+      "> 2025-03-11 16:58:31,734 [debug] Saving function: {\"runtime_name\":\"histogram-data-drift\",\"tag\":\"latest\"}\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "DeployStatus(state=ready, outputs={'endpoint': 'http://nuclio-tutorial1-admin-histogram-data-drift.default-tenant.svc.cluster.local:8080', 'name': 'tutorial1-admin-histogram-data-drift'})"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import mlrun.model_monitoring.applications.histogram_data_drift as histogram_data_drift\n",
+    "\n",
+    "custom_hist_app = project.set_model_monitoring_function(\n",
+    "    name=histogram_data_drift.HistogramDataDriftApplicationConstants.NAME,\n",
```

Comment:
Please let the reader of this notebook know that using the same function `name` is essential for the feature-analysis page to work.
It also should be made clear in the [app docstring](https://github.com/mlrun/mlrun/blob/c4ea6fd8b0308d10df5700640b01433d58107c53/mlrun/model_monitoring/applications/histogram_data_drift.py#L89).